packages/core/src/layout/InfoCard/InfoCard.tsx

```diff
@@ -0,0 +1,228 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { FC, ReactNode } from 'react';
+import {
+  Card,
+  CardActions,
+  CardContent,
+  CardHeader,
+  Divider,
+  withStyles,
+  makeStyles,
+} from '@material-ui/core';
+import ErrorBoundary from '../ErrorBoundary/ErrorBoundary';
+import BottomLink from './BottomLink';
+import { BackstageTheme } from '../../theme/theme';
+
+const useStyles = makeStyles<BackstageTheme>(theme => ({
+  header: {
+    padding: theme.spacing(2, 2, 2, 2.5),
+  },
+}));
+
+const BoldHeader = withStyles({
+  title: { fontWeight: 700 },
+  subheader: { paddingTop: '2px' },
+})(CardHeader);
+
+const CardActionsTopRight = withStyles({
+  root: {
+    display: 'inline-block',
+    paddingRight: '16px',
```

Comment:
These should be using the `theme.spacing` as well.

Reply:
Not exactly sure i did it right, but i think my latest commit should fix this.